lib/matplotlib/tests/test_backend_qt.py

```diff
@@ -33,8 +33,22 @@ def qt_core(request):
     return QtCore
 
 
+@pytest.fixture
+def platform_simulate_ctrl_c(request):
+    import signal
+    from functools import partial
+
+    if hasattr(signal, "CTRL_C_EVENT"):
+        from win32api import GenerateConsoleCtrlEvent
```

Comment:
this adds a dependency on pywin32 for tests on Windows?  (that's fine, but needs to be documented or at least made an importorskip or whatnot?)

Reply:
It looks like we install that for some other reason on appveyor, and it gets pulled in through something else in Azure.

Reply:
#20883